content/en/docs/concepts/cluster-administration/flow-control.md

```diff
@@ -0,0 +1,308 @@
+{{< /caution >}}
+
+When the volume of inbound requests assigned to a single
+PriorityLevelConfiguration is more than its permitted concurrency level, the
+`type` field of its specification determines what will happen to extra requests.
+A type of `Reject` means that excess traffic will immediately be rejected with
+an HTTP 429 (Too Many Requests) error. A type of `Queue` means that requests
+above the threshhold will be queued, and the fair queuing algorithm will be used
+to balance progress between requests based on their assigned flow distinguisher.
+
+The queuing configuration allows tuning the fair queuing algorithm for a
+priority level. Details of the algorithm can be read in the
+[KEP](https://github.com/kubernetes/enhancements/blob/master/keps/sig-api-machinery/20190228-priority-and-fairness.md),
+but in short:
+
+* Increasing `queues` reduces the rate of collisions between different flows, at
+  the cost of increased memory usage. A value of 1 here effectively disables the
+  fair-queuing logic, but still allows requests to be queued.
+
+* Increasing `queueLengthLimit` allows larger bursts of traffic to be sustained
+  without dropping any requests, at the cost of increased memory usage.
+
+* Changing `handSize` allows you to adjust the probability of collisions between
+  different flows and the overall concurrency available to a single flow in an
+  overload situation, traded off against resource consumption. 
+    {{< note >}}
+    The probability that two flows with different flow distinguishers
+    can starve each other of resources is `1 / choose(queues, handSize)`. The choice
+    of `handSize` that minimizes this probability is therefore `Queues / 2`. However,
+    much smaller values of `handSize` will in practice make such collisions very
+    unlikely while consuming less resources. A larger `handSize` also increases the
+    effective concurrency share available to a single flow in an overload situation.
+    {{< /note >}}
+
+### FlowSchema
+A FlowSchema matches some inbound requests and assigns them to a
+PriorityLevelConfiguration. Every inbound request is matched against every
+FlowSchema in turn, starting with those with highest `MatchingPrecedence` and
+working down. 
+
+{{< caution >}}
+Only the first matching FlowSchema for a given request matters.
+If multiple FlowSchemas match a single inbound request, it will be assigned
+based on the one with the highest `MatchingPrecedence`.
+{{< /caution >}}
+
+A FlowSchema matches a given request if at least one of its `rules`
+matches. A rule matches if at least one of its `subjects` *and* at least
+one of its `resourceRules` or `nonResourceRules` (depending on whether the
+incoming request is for a resource or non-resource URL) matches. 
+
+For the `name` field in subjects, and the `verbs`, `apiGroups`, `resources`,
+`namespaces`, and `nonResourceURLs` fields of resource and non-resource rules,
+the wildcard `*` may be specified to match all of the appropriate object,
```

Comment:
Is there a typo here?  "all of the appropriate object" looks odd to me.  Did you mean "all of the appropriate objects"?  I would suggest something a little more explicit, like "`*` matches any value of the relevant field", or maybe "`*` may be specified to indicate that the relevant field does not matter".  The latter is a little less accurate in the case of target namespace, because a field value of empty-string is used to signal that the request is not namespaced but matching that is stipulated by a different part of the rule.

Reply:
PTAL